tvbwidgets/core/pse/parameters.py

```diff
@@ -245,6 +247,16 @@ def _init_checkpoint(self):
     def monitor_execution(self):
         if self.update_progress is not None:
             self.update_progress()
+        else:
+            with self.progress_file_lock:
+                with open("progress_bar_status.txt", "r+") as f:
```

Comment:
The name of the file `progress_bar_status.txt` need to be the same.
Can we reuse a constant ?